src/PrivateRoute.jsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
```

Comment:
Oh no I see one last eslint-disable

Reply:
Yeah this one can stay disabled, because in this  i got the error "prop spreading is forbidden". i discussed it with ilias en tonny and for the time being it can stay like this.

Reply:
Ah okay, didn‘t know that that was this one, then it's fine :)

Reply:
@shayeedjamal But you disabled eslint for the entire file. That's not the same as disabling single rule

Reply:
Also, please disable it globaly